packages/node_modules/@webex/plugin-meetings/src/meeting/in-meeting-actions.ts

```diff
@@ -13,36 +13,73 @@ export default class InMeetingActions {
   namespace = MEETINGS;
 
   canInviteNewParticipants: boolean = null;
-  canAdmitParticipant: boolean  = null;
+
+  canAdmitParticipant: boolean = null;
+
   canLock: boolean = null;
+
   canUnlock: boolean = null;
+
   canAssignHost: boolean = null;
+
   canStartRecording: boolean = null;
+
   canPauseRecording: boolean = null;
+
   canResumeRecording: boolean = null;
+
   canStopRecording: boolean = null;
+
   canRaiseHand: boolean = null;
+
   canLowerAllHands: boolean = null;
+
   canLowerSomeoneElsesHand: boolean = null;
 
-  get() {
-    return {
-      canInviteNewParticipants: this.canInviteNewParticipants,
-      canAdmitParticipant: this.canAdmitParticipant,
-      canLock: this.canLock,
-      canUnlock: this.canUnlock,
-      canAssignHost: this.canAssignHost,
-      canStartRecording: this.canStartRecording,
-      canPauseRecording: this.canPauseRecording,
-      canResumeRecording: this.canResumeRecording,
-      canStopRecording: this.canStopRecording,
-      canRaiseHand: this.canRaiseHand,
-      canLowerAllHands: this.canLowerAllHands,
-      canLowerSomeoneElsesHand: this.canLowerSomeoneElsesHand,
-    };
-  }
-
-  set(actions) {
+  canEnableClosedCaption: boolean = null;
```

Comment:
incorrect type for all the members. All of them `boolean` but you initialised with `null` so the correct type is `null | boolean`

Reply:
maybe instead of `null | boolean` we could just have `canEnableClosedCaption?: boolean` so then it's basically undefined or a boolean?